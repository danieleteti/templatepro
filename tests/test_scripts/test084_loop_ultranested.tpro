************* Ultra Nested List of objects
{{for lev1 in ultranestedlist}}
LEV1>
{{for lev2 in lev1}}
LEV2>
Value1 = {{:lev2.Value1}}
LEV2<
{{endfor}} 
LEV1<
{{endfor}}