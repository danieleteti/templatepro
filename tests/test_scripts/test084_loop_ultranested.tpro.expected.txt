************* Ultra Nested List of objects
LEV1>
LEV2>
Value1 = Value1.1
LEV2<
LEV2>
Value1 = Value1.2
LEV2<
 
LEV1<
LEV1>
LEV2>
Value1 = Value2.1
LEV2<
LEV2>
Value1 = Value2.2
LEV2<
 
LEV1<
LEV1>
LEV2>
Value1 = Value3.1
LEV2<
LEV2>
Value1 = Value3.2
LEV2<
 
LEV1<
